### PR TITLE
Draft updates - pre IETF 116

### PR DESCRIPTION
**Pull request by nicorusti**
Changes summary:

- Terminology: changed "end-host" to "endpoint", to be more in line with the upcoming [draft-irtf-panrg-path-properties](https://datatracker.ietf.org/doc/draft-irtf-panrg-path-properties/)  
- Data plane:  Added mention of the possibility of future work to adapt the data plane to fit into an IPv6 extension (see https://github.com/scionproto/scion/discussions/4324 )
- Updated authors emails
- Small cosmetics

- - -
This change is [Reviewable](https://reviewable.io/reviews/scionassociation/scion-components_I-D/12)

### CODE REVIEW

File: draft-rustignoli-panrg-scion-components.md [hunk 1]
```diff
@@ -596,7 +598,10 @@ PANAPI (Path-Aware Networking API) {{slides-113-taps-panapi}} is being evaluated
 SCION is an inter-domain network architecture and as such its data plane does not interfere with intra-domain forwarding.
 It re-uses the existing intra-domain data and control plane to provide connectivity among its infrastructure services, border routers, and endpoints, minimizing changes to the internal infrastructure.
 This corresponds to the practice today where ASes use an intra-domain protocol of their choice (i.e., OSPF, IS-IS, MPLS, ...).
-For example, in some of the early deployments, intra-AS SCION packets are encapsulated into an IP/UDP datagram, reusing the network's existing IGP and IP-based data plane.
+
+In SCION's curret implementation and early deployments, intra-AS SCION packets are encapsulated into an IP/UDP datagram for AS-local packet delivery, reusing the network's existing IGP and IP-based data plane inside an AS.
```
Comment (cdekater) [inline draft-rustignoli-panrg-scion-components.md]: Small typo: curre**N**t (instead of curret - the N is missing)

File: draft-rustignoli-panrg-scion-components.md [hunk 2]
```diff
@@ -596,7 +598,10 @@ PANAPI (Path-Aware Networking API) {{slides-113-taps-panapi}} is being evaluated
...
+This design decision eased early deployments of SCION in IP-based networks.
+In the long term, it is not excluded that SCION's dataplane could be better integrated with IP, by implementing it as an IPv6 routing header {{rfc8200}}.
```
Comment (cdekater) [inline draft-rustignoli-panrg-scion-components.md]: Replace "dataplane" by "data plane" (to be consistent, you write "data plane" elsewhere). 
And: Are links case-sensitive? If yes, it must be {{RFC8200}} (instead of **rfc**8200).

File: draft-rustignoli-panrg-scion-components.md [hunk 3]
```diff
@@ -440,18 +440,18 @@ In conclusion, SCION is built around a unique trust model, justifying the existe
 ## Routing - Control Plane
 The SCION control plane's main purpose is to securely discover and disseminate routing information.
 Path exploration is based on path-segment construction beacons (PCBs), which are initiated by a subset of ASes and accumulate cryptographically protected path forwarding information.
-Each AS selects a few PCBs and makes them available to end hosts via its path service, part of the control plane.
+Each AS selects a few PCBs and makes them available to endpoints via its path service, part of the control plane.
 
-Overall, the Control Plane takes an unexplored topology and AS certificates as input, it then discovers the inter-domain topology and makes routing information available to end hosts.
+Overall, the Control Plane takes an unexplored topology and AS certificates as input, it then discovers the inter-domain topology and makes routing information available to endpoints.
```
Comment (cdekater) [inline draft-rustignoli-panrg-scion-components.md]: Why suddenly capitalize **C**ontrol **P**lane? In the rest of the draft it is "control plane" without capitals..

File: draft-rustignoli-panrg-scion-components.md [hunk 4]
```diff
@@ -570,13 +570,13 @@ Such extensions can be hop-by-hop (and are processed at each hop), or end-to-end
 
 - *Path validation.* SCION routers validate network paths in packets at each hop, so that they are only forwarded along paths that were authorized by all on-path ASes in the control plane. Thanks to a system of nested message authentication codes, traffic hijacking attacks are avoided.
 
-In conclusion, in comparison to today's Internet, the SCION's data plane pushes some of the responsibilities away from routers onto end hosts (such as selecting paths or reacting to failures).
+In conclusion, in comparison to today's Internet, the SCION's data plane pushes some of the responsibilities away from routers onto endpoints (such as selecting paths or reacting to failures).
```
Comment (cdekater) [inline draft-rustignoli-panrg-scion-components.md]: Replace "SCION's data plane _pushes_ some of the responsibilities away from routers _onto_ endpoints" with
"the SCION's data plane **takes** some of the responsibilities away from routers and **places them on** endpoints"

File: draft-rustignoli-panrg-scion-components.md [hunk 5]
```diff
@@ -49,33 +49,29 @@ informative:
...
-        org: ETH Zürich
+        org: SCION Association
       -
         ins: A. Perrig
         name: Adrian Perrig
         org: ETH Zürich
 
   I-D.dekater-scion-pki:
-    title: SCION Overview
+    title: SCION PKI
```
Comment (cdekater) [inline draft-rustignoli-panrg-scion-components.md]: The correct of the draft is "SCION Control-Plane PKI"

**Review by matzf — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/scionassociation/scion-components_I-D/12)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @nicorusti)

___
*[`draft-rustignoli-panrg-scion-components.md` line 604 at r1](https://reviewable.io/reviews/scionassociation/scion-components_I-D/12#-NPrN5Fk8HSE6-jWAh_L:-NPrN5Fk8HSE6-jWAh_M:ba2ji9n) ([raw file](https://github.com/scionassociation/scion-components_I-D/blob/d63094f0a8dbabc7c3393a2637abca77ea705ab4/draft-rustignoli-panrg-scion-components.md#L604)):*
> ```Markdown
> In SCION's curret implementation and early deployments, intra-AS SCION packets are encapsulated into an IP/UDP datagram for AS-local packet delivery, reusing the network's existing IGP and IP-based data plane inside an AS.
> This design decision eased early deployments of SCION in IP-based networks.
> In the long term, it is not excluded that SCION's dataplane could be better integrated with IP, by implementing it as an IPv6 routing header {{RFC8200}}.
> ```

Maybe move this to below the discussion of segment routing; then the statement is: "it doesn't work with the segment routing header, but, with a _custom_ routing header we could maybe make it work." 

I think it would be worth expanding on this topic and the challenges it poses:

The main challenges are:
- intra-domain forwarding needs to be revisited (either make intra-domain routing infra IPv6+SCION aware, use other routing headers on top for local forwarding (SRv6), or encapsulate).
- addressing: SCION endpoint needs AS information to query path to destination; IPv6 addresses don't contain this information. Either keep separate SCION addresses, assign special address ranges that encode AS number, or design (secure) mapping from address ranges to ASes.
___
*[`draft-rustignoli-panrg-scion-components.md` line 616 at r1](https://reviewable.io/reviews/scionassociation/scion-components_I-D/12#-NPrMoaQ9wApBrfAJPs9:-NPrMoaQ9wApBrfAJPsA:b-m01xlm) ([raw file](https://github.com/scionassociation/scion-components_I-D/blob/d63094f0a8dbabc7c3393a2637abca77ea705ab4/draft-rustignoli-panrg-scion-components.md#L616)):*
> ```Markdown
> 
> 
> In conclusion, thanks to its data plane, SCION achieves properties that are difficult to achieve when exclusively extending existing protocols.
> ```

This seems to contradict the new statement on an IPv6 extension now. Perhaps _replace_ this with an extended discussion of the IPv6 routing header idea.

**Review by nicorusti — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/scionassociation/scion-components_i-d/12)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @matzf)

___
*[`draft-rustignoli-panrg-scion-components.md` line 604 at r1](https://reviewable.io/reviews/scionassociation/scion-components_i-d/12#-NPrN5Fk8HSE6-jWAh_L:-NQC6dC_6jZ3JzaZIFy5:b-gl5lhl) ([raw file](https://github.com/scionassociation/scion-components_i-d/blob/d63094f0a8dbabc7c3393a2637abca77ea705ab4/draft-rustignoli-panrg-scion-components.md#L604)):*
<details><summary><i>Previously, matzf (Matthias Frei) wrote…</i></summary><blockquote>

Maybe move this to below the discussion of segment routing; then the statement is: "it doesn't work with the segment routing header, but, with a _custom_ routing header we could maybe make it work." 

I think it would be worth expanding on this topic and the challenges it poses:

The main challenges are:
- intra-domain forwarding needs to be revisited (either make intra-domain routing infra IPv6+SCION aware, use other routing headers on top for local forwarding (SRv6), or encapsulate).
- addressing: SCION endpoint needs AS information to query path to destination; IPv6 addresses don't contain this information. Either keep separate SCION addresses, assign special address ranges that encode AS number, or design (secure) mapping from address ranges to ASes.
</blockquote></details>

I moved the text after the SR discussion. Regarding expanding on the topic, I think it makes sense, but I don't have too much time to prepare a discussion on this for this draft iteration. 

I'll keep this topic as future work and keep it in mind in https://github.com/scionassociation/scion-components_I-D/issues/13 

Perhaps this could also be an interesting discussion to bring to an IETF WG at some point.. 
___
*[`draft-rustignoli-panrg-scion-components.md` line 616 at r1](https://reviewable.io/reviews/scionassociation/scion-components_i-d/12#-NPrMoaQ9wApBrfAJPs9:-NQC6VPp2mfpcMPUas_Q:b-4ngan6) ([raw file](https://github.com/scionassociation/scion-components_i-d/blob/d63094f0a8dbabc7c3393a2637abca77ea705ab4/draft-rustignoli-panrg-scion-components.md#L616)):*
<details><summary><i>Previously, matzf (Matthias Frei) wrote…</i></summary><blockquote>

This seems to contradict the new statement on an IPv6 extension now. Perhaps _replace_ this with an extended discussion of the IPv6 routing header idea.
</blockquote></details>

You are right.. Indeed we are saying that potentially the data plane could be rethought, I'll remove this sentence.
